generator/standalone_main.cc

```diff
@@ -30,6 +30,7 @@
 #include <google/protobuf/compiler/command_line_interface.h>
```

Comment:
Also to double check my understanding, is there were the doc generation logic lives?